Python/Star_Pattern/Triangle_Star_Pattern.py

```diff
@@ -0,0 +1,29 @@
+def triangle(n):

```

Comment:
Whole code is copy form [here](https://www.geeksforgeeks.org/programs-printing-pyramid-patterns-python/)